cpp_test_suite/event/stateless_sub.cpp

```diff
@@ -96,7 +96,7 @@ int main(int argc, char **argv)
 // Wait for connection and event
 //
 
-		Tango_sleep(40);
+		Tango_sleep(50);
```

Comment:
Strangely enough this test always passes in RELEASE mode but not in DEBUG. I suspect some optimizations take place and therefore the code is fast enough to catch last non-error callback.